components/apimgt/org.wso2.carbon.apimgt.output.adapter.http/src/main/java/org/wso2/carbon/apimgt/output/adapter/http/internal/ds/ServiceReferenceHolder.java

```diff
@@ -0,0 +1,42 @@
+/*
+ *  Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+package org.wso2.carbon.apimgt.output.adapter.http.internal.ds;
+
+import org.wso2.carbon.apimgt.impl.recommendationmgt.AccessTokenGenerator;
+
+public class ServiceReferenceHolder {
```

Comment:
This isn't a service reference holder anymore right? Also, this doesn't have to be inside internal package since this isn't an osgi activator class

Reply:
Fixed